tests/integration/connect-test.js

```diff
@@ -93,3 +93,17 @@ test('lifecycle hooks are still invoked', function(assert) {
 
   this.set('name', 'Dustin');
 });
+
+test('connecting dispatchToActions only', function(assert) {
+  assert.expect(1);
+  const dispatchToActions = () => {};
+
+  this.register('component:test-component', connect(null, dispatchToActions)(Ember.Component.extend({
+    init() {
+      this._super(...arguments);
+      assert.ok(true, 'should be able to connect components passing `null` to stateToComputed');
+    }
```

Comment:
@dustinfarris can we also add something here for undefined (assuming both work obviously - just want to keep us in check going forward)